pydis_site/apps/content/urls.py

```diff
@@ -29,14 +29,35 @@ def __get_all_files(root: Path, folder: typing.Optional[Path] = None) -> list[st
     return results
 
 
-def get_all_pages() -> typing.Iterator[dict[str, str]]:
+DISTILL_RETURN = typing.Iterator[dict[str, str]]
+
+
+def get_all_pages() -> DISTILL_RETURN:
     """Yield a dict of all page categories."""
     for location in __get_all_files(Path("pydis_site", "apps", "content", "resources")):
         yield {"location": location}
 
 
+def get_all_tags() -> DISTILL_RETURN:
+    """Return all tag names and groups in static builds."""
+    groups = {None}
```

Comment:
A comment couldn't hurt here as to why we start with `{None}`. I can understand it, as Chris has lovingly explained it to me.

Reply:
3940e34b